bee-node/build.rs

```diff
@@ -36,3 +79,89 @@ fn main() -> Result<(), BuildError> {
 
     Ok(())
 }
+
+#[cfg(feature = "dashboard")]
+mod dashboard {
+    use super::*;
+
+    use serde::Deserialize;
+    use zip::ZipArchive;
+
+    use std::{
+        io::{self, BufReader, Cursor},
+        path::Path,
+    };
+
+    const ASSET_PREFIX: &str = "node-dashboard-bee-";
+    const RELEASES_URL: &str = "https://api.github.com/repos/iotaledger/node-dashboard/releases/latest";
```

Comment:
Maybe we should pin the version of the dashboard release so that we don't get any surprises in the future? That way we also wouldn't need the `LatestReleaseAssets` logic.

Reply:
I would just hard code it as a `const &str`, same for the SHA sum, this will only very rarely change. You can just get it directly from here: `https://github.com/iotaledger/node-dashboard/releases/download/v1.0.0/node-dashboard-bee-1.0.0.zip`

Reply:
Sure, I can certainly do that.